annotation/app/models.py

```diff
@@ -332,3 +359,51 @@ def get_stats(self):
             "time_finish": self.task.stats.updated,
             "agreement_score": self.agreement_score,
         }
+
+
+class DocumentLinks(Base):
+    __tablename__ = "document_links"
+
+    original_revision = Column(VARCHAR)
+    original_file_id = Column(INTEGER)
+    original_job_id = Column(INTEGER)
+
+    similar_revision = Column(VARCHAR)
+    similar_file_id = Column(INTEGER)
+    similar_job_id = Column(INTEGER)
```

Comment:
Can we make this as composite key?

Reply:
I actually do [here](https://github.com/epam/badgerdoc/pull/71/files/ee263455b1a742f1789eaaf23364ffa7faa1fe39#diff-3ac0632a865326abdf7db8425d85b915e0ce5fcff9516381b0cfc6e7a276a036R382-R384). SQLAlchemy magic =) 
Because these columns are FKs to a composite key as well we have to mark them like I did, otherwise SQLAlchemy assumes we use only one column referring to a composite key, which leads us to complains about PK constraint in `annotated_doc` by ORM